webrender/src/frame.rs

```diff
@@ -340,12 +340,12 @@ impl Frame {
         let layer_rect = LayerRect::new(LayerPoint::zero(),
                                         LayerSize::new(content_size.width + clip.origin.x,
                                                        content_size.height + clip.origin.y));
-        context.builder.push_layer(layer_rect,
-                                   &ClipRegion::simple(&layer_rect),
-                                   LayerToScrollTransform::identity(),
-                                   pipeline_id,
-                                   current_scroll_layer_id,
-                                   &[]);
+        context.builder.push_stacking_context(layer_rect,
+                                              &ClipRegion::simple(&layer_rect),
+                                              LayerToScrollTransform::identity(),
```

Comment:
Not directly related to this PR, but is it correct to still call it `LayerToScrollTransform`?
AFAIK, the SC's/items are attached to their reference frames, but the transformation we pass here is relative to the parent stacking context (not necessarily the scroll layer).